Support/Testing/Travis/install.py

```diff
@@ -62,9 +62,6 @@
 
 if target in android_toolchains:
     # Android builds get the toolchain from AOSP and use platform 21 by default.
-    android_platform = os.getenv('ANDROID_PLATFORM', "21")
-    check_call('./Support/Scripts/prepare-android-toolchain.sh "%s:%s"' %
```

Comment:
That's gonna break Travis for the android builds that haven't been moved to CircleCI yet, but I don't really care :p

Reply:
It'll break Travis next year, so who cares?